woocommerce-gateway-ebanx/includes/class-wc-ebanx-gateway.php

```diff
@@ -49,6 +49,8 @@ public function __construct()
 		$this->names = $this->get_billing_field_names();
 
 		$this->merchant_currency = strtoupper(get_woocommerce_currency());
+
+		$this->language = $this->getTransactionAddress('country');
```

Comment:
The other place it was being called was "is_available" which also is a place the user is not logged in.
Checked the function and it validates exactly this on it's first 3 lines.